terminal.py

```diff
@@ -554,12 +554,7 @@ def log_settings() -> None:
     logger.info("SETTINGS: %s ", str(settings_dict))
 
 
-def run_scripts(
```

Comment:
@tehcoderer you remove my whole PR https://github.com/GamestonkTerminal/GamestonkTerminal/pull/1455.

When adding files is better to do it manually one by one.